rest_api/VulnMgmtService.adoc

```diff
@@ -0,0 +1,3129 @@
+// Auto-generated by scripts. Do not edit.
+:_mod-docs-content-type: ASSEMBLY
+
+
+
+[id="VulnMgmtService"]
+= VulnMgmtService
+
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+
+[id="VulnMgmtServiceVulnMgmtExportWorkloads"]
+== VulnMgmtExportWorkloads
+
+`GET /v1/export/vuln-mgmt/workloads`
+
+Streams vulnerability data upon request. Each entry consists of a deployment and the associated container images.
+
+=== Description
+
+The response is structured as: {\"result\": {\"deployment\": {...}, \"images\": [...]}} ... {\"result\": {\"deployment\": {...}, \"images\": [...]}}
+
+
+
+
+
+=== Parameters
+
+
+
+
+
+==== Query Parameters
+
+[cols="2,3,1,1,1"]
+|===
+|Name| Description| Required| Default| Pattern
+
+| timeout
+| Request timeout in seconds.
+| -
+| null
+|
+
+| query
+| Query to constrain the deployments for which vulnerability data is returned. The queries contain pairs of &#x60;Search Option:Value&#x60; separated by &#x60;+&#x60; signs. For HTTP requests the query should be quoted. For example &gt; curl \&quot;$ROX_ENDPOINT/v1/export/vuln-mgmt/workloads?query&#x3D;Deployment%3Ascanner%2BNamespace%3Astackrox\&quot; queries vulnerability data for all scanner deployments in the stackrox namespace. See https://docs.openshift.com/acs/operating/search-filter.html for more information.
+| -
+| null
+|
+
+|===
+
+
+=== Return Type
+
+<<Stream_result_of_v1VulnMgmtExportWorkloadsResponse>>
```

Comment:
Same problem as noted with `<<object>>` and the clickable link that is created, but no actual destination in the link.
![Screenshot 2024-09-25 at 3 23 50 PM](https://github.com/user-attachments/assets/8bf56625-b76e-4abc-9567-18939ef8740f)